common/messaging/kafkaConfig.go

```diff
@@ -22,26 +22,20 @@ package messaging
 
 import (
 	"fmt"
+
+	"github.com/uber/cadence/common/auth"
 )
 
 type (
 	// KafkaConfig describes the configuration needed to connect to all kafka clusters
 	KafkaConfig struct {
-		TLS            TLS                      `yaml:"tls"`
+		TLS            auth.TLS                 `yaml:"tls"`
 		Clusters       map[string]ClusterConfig `yaml:"clusters"`
 		Topics         map[string]TopicConfig   `yaml:"topics"`
 		ClusterToTopic map[string]TopicList     `yaml:"cadence-cluster-topics"`
 		Applications   map[string]TopicList     `yaml:"applications"`
 	}
 
-	// TLS describe the Kafka TLS configuration
-	TLS struct {
-		Enabled    bool   `yaml:"enabled"`
-		CertFile   string `yaml:"certFile"`
-		KeyFile    string `yaml:"keyFile"`
-		BundleFile string `yaml:"bundleFile"`
```

Comment:
This will be a breaking change for users who use TLS in Kafka. 
@wxing1292 Do you know if any customer is using this? Can we notice them and get this change in?

Otherwise for safety, I would keep it for kafka, and use the new auth.TLS for Cassandra and everything else. 
Or just use the old name "bundleFile"

Reply:
talked to our customer, they are ok